contracts/RegistryApp.sol

```diff
@@ -40,20 +47,17 @@ contract RegistryApp is App {
      * Remove an entry from the registry.
      * @param _id The ID of the entry to remove
      */
-    function remove(
-        bytes32 _id
-    ) public auth(REMOVE_ENTRY_ROLE) {
+    function remove(bytes32 _id) isInitialized public auth(REMOVE_ENTRY_ROLE) {
         delete entries[_id];
```

Comment:
Should only delete if entry wasn't in the registry

Reply:
I guess you mean "was", right?

Reply:
Should we do then the same check for `add`?